app/helpers/staff_helper.rb

```diff
@@ -0,0 +1,21 @@
+module StaffHelper
+
+  def allow_rating?(proposal)
+    program_mode? || !proposal.has_speaker?(current_user)
+  end
+
+  def show_ratings?(rating)
+    program_mode? || rating.persisted?
+  end
+
+  def show_proposal?(proposal)
+    program_mode? || !proposal.has_speaker?(current_user)
+  end
+
+  def program_tracker
+    if program_mode?
+      hidden_field_tag(:program, true)
+    end
+  end
```

Comment:
I'm wondering how to consolidate this logic into a policy. allow_rating? and show_proposal? seem to duplicate ProposalPolicy#review? though without the staff check. show_ratings? is more of a pure view concern, and program_mode? is tied to the controller/view scope and wouldn't work inside a policy. A combination of both might work here, something like: 
```
def allow_rating?(proposal)
  program_mode? || policy(proposal).rate?
end
```